### ISSUE TITLE
Updated today and now no networks seen?

### ISSUE THREAD

**Issue by deadrubberboy** [schmittx/home-assistant-eero#90]
Updated to 1.6.0 this morning. First just the “erro profiles” stopped reporting (I use them with your plug for presence detection). Then the whole thing went away and all entities gone. 

Restored backup I took before hand and all is well.  Will wait on updating

**Comment by onkelfarmor** [comment]
I see the same but also before updating, I think it happened with the latest HA update.

![IMG_0101](https://github.com/user-attachments/assets/3f9110a6-88c3-4a07-bf6b-55aede17f732)

**Comment by iiTranquility** [comment]
Same issue, no real logs, only just started using it yesterday, non amazon account

**Comment by neilmartin83** [comment]
Same here - got some log output:

In the HA log after a restart following updating 1.5 to 1.6

```
2024-07-30 17:43:53.445 ERROR (SyncWorker_27) [custom_components.eero.api] Parsing response was unsuccessful with status code: 403
2024-07-30 17:45:12.485 DEBUG (SyncWorker_15) [custom_components.eero.api] Calling API with method: GET and URL: /2.2/account
2024-07-30 17:45:12.710 DEBUG (SyncWorker_15) [custom_components.eero.api] Calling API with method: GET and URL: /2.2/networks/7331731
2024-07-30 17:45:13.569 DEBUG (SyncWorker_15) [custom_components.eero.api] Calling API with method: GET and URL: /2.2/networks/7331731/thread
2024-07-30 17:45:13.766 DEBUG (SyncWorker_15) [custom_components.eero.api] Calling API with method: GET and URL: /2.2/networks/7331731/backup_access_points
2024-07-30 17:45:13.967 ERROR (SyncWorker_15) [custom_components.eero.api] Parsing response was unsuccessful with status code: 403
2024-07-30 17:45:13.968 DEBUG (SyncWorker_15) [custom_components.eero.api] 
- EeroException
- Code: 403
- Error: error.premium.user_not_subscribed
- Server time: 2024-07-30T16:45:13.888Z
2024-07-30 17:45:24.970 DEBUG (SyncWorker_31) [custom_components.eero.api] Calling API with method: GET and URL: /2.2/account
2024-07-30 17:45:25.595 DEBUG (SyncWorker_31) [custom_components.eero.api] Saving response: /config/custom_components/eero/api/responses/_2_2_account.json
2024-07-30 17:45:25.597 DEBUG (SyncWorker_31) [custom_components.eero.api] Saving response: /config/custom_components/eero/api/responses/update_data.json
2024-07-30 17:45:25.601 DEBUG (MainThread) [custom_components.eero] Finished fetching Eero (Home Assistant) data in 0.634 seconds (success: True)
```

Sanitised API response JSON files:

[update_data.json](https://github.com/user-attachments/files/16431097/update_data.json)
[_2_2_account.json](https://github.com/user-attachments/files/16431098/_2_2_account.json)

**Comment by MasterDevwi** [comment]
+1. Reverting to 1.5.0 fixed it.

**Comment by strickersuk** [comment]
+1 More, rolling back to 1.5.0 worked a treat.

**Comment by Stromber03** [comment]
How to rolling back to 1.5.0?

**Comment by strickersuk** [comment]
Remove the integration, then remove the HACS download. Restart, re add the HACS and select 1.5.0 to download, make sure you wait until the 1.5.0 isn't grey. Restart again and add the integration.

**Comment by semyonchetvertnyh** [comment]
@strickersuk @Stromber03 rolling back can be done without removing the integration. Just cick "Redownload" at the HACS integration page and select 1.5.0 version or [follow the instructions](https://experimental.hacs.xyz/docs/use/entities/update/#install-service) at HACS docs.

**Comment by strickersuk** [comment]
Thanks, I think it was me being impatient and not waiting for 1.5.0 to go dark from being greyed out when selecting.

**Comment by michaelthompson1991** [comment]
For me the change version doesn’t even come up

**Comment by mikefrantz** [comment]
Is there a way to reload the Eero Home Assistant Integration in an automation if things become unavailable? I can do it manulally but would like to know if i can use something like this? If so where can I find the device ID? 
action:
  - data: {}
    action: homeassistant.reload_config_entry
    target:
      device_id:

**Comment by markwiese** [comment]
I rolled back to version 1.5 but it didn't resolve the issue. I still see no networks listed.

**Comment by michaelthompson1991** [comment]
I'm still on 1.5.0 I believe and it seems to be working

**Comment by markwiese** [comment]
I also tried rolling back to 1.4.5, no luck. I've only just switched back to using Eero routers this week. I had an Eero mesh some months ago and the integration was working for me, then. Perhaps it's something to do with replacing my network with one of the same name (the name of the network in the Eero app) or that I now manage 2 networks in the app (mine and my partner's).

**Comment by markwiese** [comment]
OK, my apologies, it's been so long since I initially started using this add-on that I'd forgotten you can't use your Amazon account. I literally don't remember originally setting up an extra (non-Amazon) admin account for this, years ago, but I must have. Anyway, did the same, again, and my network showed up. Apologies for the spam.

**Comment by markwiese** [comment]
Incidentally, after getting the integration working with version 1.5.0, I tried updating to 1.6.0, and all seems fine 👍

**Comment by schmittx** [comment]
This should be fixed in [1.6.1](https://github.com/schmittx/home-assistant-eero/releases/tag/1.6.1), please update and confirm if the issue is resolved.
